src/views/open_order_list_view.js

```diff
@@ -0,0 +1,167 @@
+import Backbone from 'backbone';
+import OpenOrderView from '../views/open_order_view'
+
+import OpenOrder from '../models/open_order';
+
+const OpenOrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+    this.listenTo(this.model, 'update', this.render);
+    this.listenTo(this.bus, 'quote_symbols', this.symbolDropdown);
+    this.listenTo(this.bus, 'quote_change', this.checkOpenOrders);
+
+    // list for current quote list
+    this.listenTo(this.bus, 'current_quote_list', this.getQuoteList);
+  },
+  getQuoteList(quoteList) {
+    this.quoteList = quoteList;
```

Comment:
Instead of using an event on the bus to get the `quoteList`, why not pass it in as an extra parameter when instantiating the `OpenOrderListView`? Building an event for it is overkill.